lib/kafo/kafo_configure.rb

```diff
@@ -108,6 +115,7 @@ def execute
         store_params(temp_config_file)
       else
         store_params
+        self.class.scenario_manager.link_last_scenario(self.class.config_file) if defined?(CONFIG_DIR)
```

Comment:
could you extract `defined?(CONFIG_DIR)` to some scenario manager method? it's used on multiple places.